### DISCUSSION TITLE
configuring home directory |need help|

### DISCUSSION THREAD

**Discussion by Cruising0904** [nvim-telekasten/telekasten.nvim#220] [Q&A] [answered]
I tested it months before failed it and now I am trying to get it to work. But setting up directory I am not following it keeps making opposite from how I wanted. When I call telekasten to make a new note it makes directory "wiki" relative path (home = "wiki"), at home directory it just makes zettelekasten directory. However I would like to manage all my md files in .dotfiles/wiki directory like I did using vimwiki. 
Is it possible to custom root directory? or is it just a bug or I am just trolling..

**Comment by Cruising0904** [comment]
I am using linux

I looked up wiki there is only for windows

**Comment by lambtho12** [comment, marked as answer]
What you want is something like that in your `init.lua`:

```lua
 -- Zettelkasten directory
  local home = vim.fn.expand("~/.dotfiles/wiki")
  local journal = home .. "/journal"

  require("telekasten").setup({
    -- Define directories
    home = home,
    dailies = journal .. "/daily",
    weeklies = journal .. "/weekly",
    templates = home .. "/templates",
    image_subdir = home .. "/img",
  })
```

The line `local home = vim.fn.expand("~/.dotfiles/wiki")`  is where you define your actual wiki home. As you see, all other options are just subdirectory of that root dir.

**Reply by lambtho12** [reply]
See the help within nvim for more details about the different options `:h Telekasten`. The wiki on github is not really maintained at the moment.

**Reply by Cruising0904** [reply]
thank you very much! You saved my whole day :+1:

**Reply by lambtho12** [reply]
Awesome!
